packages/common/src/types/index.ts

```diff
@@ -57,6 +57,27 @@ export type Directory = {
   sourceId: string;
 };
 
+export type Template = {
+  name: TemplateType;
+  niceName: string;
+  shortid: string;
+  url: string;
+  main: boolean;
+  color: string;
+  backgroundColor: () => string | undefined;
```

Comment:
@SaraVieira This can just be

```suggestion
  backgroundColor?: () => string;
```

Reply:
I don't think that should be the case here. There's a difference `backgroundColor?` and `backgroundColor: () => string | undefined`.